xds/src/main/java/com/linecorp/centraldogma/xds/internal/ControlPlanePlugin.java

```diff
@@ -148,6 +149,12 @@ private void init0(PluginInitContext pluginInitContext) {
                                                    .useBlockingTaskExecutor(true)
                                                    .build();
         sb.route().build(grpcService);
+        final GrpcService xdsApplicationService =
+                GrpcService.builder()
+                           .addService(new XdsGroupService(
+                                   projectManager, pluginInitContext.commandExecutor()))
+                           .enableHttpJsonTranscoding(true).build();
+        sb.service(xdsApplicationService, pluginInitContext.authService());
```

Comment:
Question) Just so I understand the current state, IIUC only admins have access to internal projects. Hence, am I understanding correctly that only admins will be able to use `XdsGroupService` for now?

Reply:
Anyone can use the `XdsGroupService` for now as I talked before.
https://github.com/line/centraldogma/pull/981/files#diff-6530b293347a15e8133c26cfb4221f0b565c32f0a6cccb8c1d44c0d78108c8bfR62

After modifying the authorization system similar to GitHub, anyone can create a group, and the one who created the group can add anyone or tokens to the group who will have the write permission for the resources in the group.

Reply:
I see, I must have glanced through the diffset. I understood that for now, anyone can create/delete anyone's group 👍